src/test/java/seedu/address/logic/parser/DeleteSupplierCommandParserTest.java

```diff
@@ -18,11 +18,11 @@
  */
 public class DeleteSupplierCommandParserTest {
 
-    private DeleteCommandParser parser = new DeleteCommandParser();
+    private DeleteSupplierCommandParser parser = new DeleteSupplierCommandParser();
 
     @Test
     public void parse_validArgs_returnsDeleteCommand() {
-        assertParseSuccess(parser, "1", new DeleteSupplierCommand(INDEX_FIRST_SUPPLIER));
+        assertParseSuccess(parser, "1", new DeleteSupplierCommand(INDEX_FIRST_GOOD));
```

Comment:
Is it supposed to be INDEX_FIRST_SUPPLIER for the DeleteSupplierCommand?

Reply:
Fixed. Got a bit too excited during my refactoring.